diff --git a/src/helpers/Monitor.cpp b/src/helpers/Monitor.cpp
index e6117ab2..2b0450b5 100644
--- a/src/helpers/Monitor.cpp
+++ b/src/helpers/Monitor.cpp
@@ -829,10 +829,9 @@ bool CMonitor::applyMonitorRule(Config::CMonitorRule&& pMonitorRule, bool force)
     // Needed in case we are switching from a custom modeline to a standard mode
     m_customDrmMode = {};
     m_currentMode   = nullptr;
 
-    const auto initialFormat = m_drmFormat != DRM_FORMAT_INVALID ? m_drmFormat : DRM_FORMAT_XRGB8888;
-    m_output->state->setFormat(initialFormat);
+    m_output->state->setFormat(DRM_FORMAT_XRGB8888);
     m_prevDrmFormat = m_drmFormat;
-    m_drmFormat     = initialFormat;
+    m_drmFormat     = DRM_FORMAT_XRGB8888;
     m_output->state->resetExplicitFences();
 
     if (Env::isTrace()) {
