diff --git a/src/desktop/state/FocusState.cpp b/src/desktop/state/FocusState.cpp
index 5dd22f94..c4a1f6a0 100644
--- a/src/desktop/state/FocusState.cpp
+++ b/src/desktop/state/FocusState.cpp
@@ -167,9 +167,6 @@ void CFocusState::rawWindowFocus(PHLWINDOW pWindow, eFocusReason reason, SP<CWLS
         return;
     }
 
-    if (PMONITOR && !pWindow->m_pinned)
-        rawMonitorFocus(PMONITOR);
-
     const auto PLASTWINDOW                    = m_focusWindow.lock();
     m_focusWindow                             = pWindow;
     pWindow->m_workspace->m_lastFocusedWindow = pWindow;
